tools/make/lib/docs/api.mk

```diff
@@ -25,13 +41,33 @@ include $(TOOLS_MAKE_LIB_DIR)/docs/typescript.mk
 
 # RULES #
 
+#/
+# Clones the specified stdlib version for documentation building.
+#
+# @example
+# make clone-stdlib-version
+#
+# @example
+# make clone-stdlib-version STDLIB_BRANCH=v0.2.0
+#/
+clone-stdlib-version:
```

Comment:
It is a bit odd to place this target here, but I also don't immediately have a better idea, so fine to leave as is for now.